src/Microsoft.Health.Dicom.Functions/Microsoft.Health.Dicom.Functions.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="4.0.1" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.4.3" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.2" />
-    <PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="$(SdkPackageVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="3.1.16" />
```

Comment:
`dotnet publish` becomes confused using the default `publish` command with `TargetFrameworks`, but without it the `$(SdkPackageVersion)` won't resolve nicely by default.

Reply:
Unfortunately yes, but I can refactor this slightly to make it perhaps more appealing